examples/advanced/version.tf

```diff
@@ -2,11 +2,11 @@ terraform {
   required_version = ">= 1.3.0, <1.6.0"
```

Comment:
please update this to `required_version = ">= 1.3.0, <1.7.0"` in preperation for migration to opentofu